### PR TITLE
[EN-2087] Add bridge tests

### PR DESCRIPTION
**Pull request by ilya-shablovsky**

### CODE REVIEW

File: Bridge/Bridge.csproj [hunk 1]
```diff
@@ -312,7 +312,7 @@
     <Compile Include="shared\System\IO\EndOfStreamException.cs" />
     <Compile Include="shared\System\IO\File.cs" />
     <Compile Include="shared\System\IO\FileMode.cs" />
-    <Compile Include="System\IO\FileStream.cs" />
+    <Compile Include="System\IO\filestream.cs" />
```
Comment (MakeevAlexey) [inline Bridge/Bridge.csproj:315]: Looks weird, why?

File: Bridge/Bridge.csproj [hunk 2]
```diff
@@ -563,9 +563,9 @@
     <Compile Include="System\Version.cs" />
   </ItemGroup>
   <ItemGroup>
-    <Content Include="Resources\.auxiliary\AssemblyEnd.js" />
-    <Content Include="Resources\.auxiliary\Header.js" />
-    <Content Include="Resources\.auxiliary\AssemblyBegin.js" />
+    <Content Include="Resources\.auxiliary\assemblyend.js" />
```
Comment (MakeevAlexey) [inline Bridge/Bridge.csproj:566]: Please be consistent across all C#/JS filenames

File: Compiler/Translator/Utils/Roslyn/DiscardReplacer.cs [hunk 3]
```diff
@@ -29,7 +29,14 @@ public SyntaxNode Replace(SyntaxNode root, SemanticModel model, Func<SyntaxNode,
...
                         var si = model.GetSymbolInfo(arg.Expression);
-                        return si.Symbol == null || si.Symbol.Kind == SymbolKind.Discard;
+                        try
+                        {
+                            return si.Symbol == null || si.Symbol.Kind == SymbolKind.Discard;
+                        }
+                        catch (Exception)
+                        {
+                            return true;
+                        }
```
Comment (MakeevAlexey) [inline Compiler/Translator/Utils/Roslyn/DiscardReplacer.cs:39]: afaik, it was a bug on mac. please put a comment, that it's inconsistency of bridge behaviour on different platform(runtimes) with different. this is invalueable information

File: LunaToolbox/.eslintrc.js [hunk 4]
```diff
@@ -1,26 +1,90 @@
 module.exports = {
-    env: {
-        browser: true,
-        commonjs: true,
-        es2021: true,
-    },
+    root: true,
```
Comment (MakeevAlexey) [inline LunaToolbox/.eslintrc.js:2]: i see linter but don't see linter task on CI, am i blind or just you just forgot?

File: LunaToolbox/lib/jobs/test.js [hunk 5]
```diff
@@ -0,0 +1,244 @@
+const puppeteer = require( 'puppeteer' );
```
Comment (MakeevAlexey) [inline LunaToolbox/lib/jobs/test.js:1]: As we plan to have different tools in our toolbox, please make a separatee hierarchy for bridge tools. for now filepath is not verbose

File: LunaToolbox/lib/jobs/config.js [hunk 6]
```diff
@@ -66,24 +66,36 @@ async function getConfig() {
     return JSON.parse( configDataRaw );
 }
 
-async function validateConfig() {
+function getConfigCI() {
+    return {
+        bridgePath: path.join( __dirname, '../../..' ),
```
Comment (MakeevAlexey) [inline LunaToolbox/lib/jobs/config.js:71]: is it ok for both OS?

**Review by MakeevAlexey — CHANGES_REQUESTED** [review]
Also for the future of this tool, we need to add "man" command with all available commands and descriptions

File: Compiler/Translator/Utils/Roslyn/DiscardReplacer.cs (hunk 3 above)
Comment (ilya-shablovsky) [reply Compiler/Translator/Utils/Roslyn/DiscardReplacer.cs:42]: Done.

File: LunaToolbox/lib/jobs/config.js (hunk 6 above)
Comment (ilya-shablovsky) [reply LunaToolbox/lib/jobs/config.js:71]: In fact, this option is should be used only for CI

File: Bridge/Bridge.csproj (hunk 2 above)
Comment (ilya-shablovsky) [reply Bridge/Bridge.csproj:566]: Fixed.

File: Tests/Runner/Batch2/Bridge.Test/bridge.clienttest.batch2.runner.js [hunk 7]
```diff
@@ -117,7 +117,7 @@ Bridge.assembly("Bridge.Test.Bridge.ClientTest.Batch2", function ($asm, globals)
             GetContext: function () {
                 var $t;
                 if (this.context == null) {
-                    this.context = ($t = new Bridge.Test.Runtime.FixtureContext(), $t.Project = "Batch2", $t.ClassName = "Bridge.ClientTest.Batch2.BridgeIssues.Bridge1385", $t.File = "Batch2\\BridgeIssues\\N1385.cs", $t);
+                    this.context = ($t = new Bridge.Test.Runtime.FixtureContext(), $t.Project = "Batch2", $t.ClassName = "Bridge.ClientTest.Batch2.BridgeIssues.Bridge1385", $t.File = "file:///Users/ilyashablovsky/Documents/Projects/Bridge/Tests/Batch2/BridgeIssues/N1385.cs", $t);
```
Comment (MakeevAlexey) [inline Tests/Runner/Batch2/Bridge.Test/bridge.clienttest.batch2.runner.js:120]: cleanup personal pathes

**Review by MakeevAlexey — CHANGES_REQUESTED** [review]
check once again please, what can be gitignored (.generated)
And prepare it to QA

**Pull request merged**